pyodide_lock/spec.py

```diff
@@ -91,3 +101,151 @@ def check_wheel_filenames(self) -> None:
                 for name, errs in errors.items()
             )
             raise ValueError(error_msg)
+
+    def add_wheels(  # noqa: C901
+        self,
+        wheel_files: list[Path],
+        base_path: Path | None = None,
+        base_url: str = "",
+    ) -> None:
+        """Add a list of wheel files to this pyodide-lock.json
+
+        Args:
```

Comment:
We generally use [numpy style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html#parameters)